common/app/views/fragments/atoms/ampYoutube.scala.html

```diff
@@ -1,4 +1,5 @@
-@(media: model.content.MediaAtom,  displayCaption: Boolean, mainMedia: Boolean)(implicit request: RequestHeader)
+@import model.content.MediaWrapper
+@(media: model.content.MediaAtom,  displayCaption: Boolean, mediaWrapper: Option[MediaWrapper])(implicit request: RequestHeader)
```

Comment:
We both use full class path and import in this 2 lines? (ie: `model.content.MediaAtom` and `import model.content.MediaWrapper`)
Can we stick to a single one? I vote for the full class style but it is up to you